### PR TITLE
Feat routes

### PR DESCRIPTION
**Pull request by Kothalkarpriya**
I have added routing for all the present pages.

### CODE REVIEW

File: src/App.jsx [hunk 1]
```diff
@@ -1,10 +1,21 @@
 import "./App.css";
-import { Home } from "./pages";
+import { Navbar, Footer } from "./components/component";
```
Comment (Nagashree01) [inline src/App.jsx:2]: you can remove this as it is not used here

File: src/App.jsx [hunk 2]
```diff
@@ -1,10 +1,21 @@
...
+import { Home,WishList, ProductListing, Cart, Login, Signup } from "./pages";
+import { Routes, Route } from "react-router-dom";
 
 function App() {
   return (
     <div>
-      <Home />
+    <Navbar />
+      <Routes>
+        <Route path="/" element={<Home />} />
```
Comment (Nagashree01) [inline src/App.jsx:11]: you can put all these routes in another separate file.

**Review by Nagashree01 — APPROVED** [review]
Just one suggestion. Code is clean. Good to merge

File: src/App.jsx (hunk 2 above)
Comment (Kothalkarpriya) [reply src/App.jsx:11]: Ok I will do it

**Review by iamsaurav28 — APPROVED** [review]
well written code good to merge.

**Pull request merged**